scripts/strict-null-checks-ci.ts

```diff
@@ -0,0 +1,247 @@
+/**
+ * TODO (STENCIL-446): Remove this script once `strictNullChecks` is enabled
+ */
+
+/**
+ * A script for formatting a Markdown report for CI on the number of strictNullChecks errors we're
+ * seeing on the current branch vs on main. The report also includes some info about the most
+ * error-filled files, as well as the errors we see most often.
+ */
+
+import fs from 'fs';
+
+/**
+ * This interface is copied (a bit) from here:
+ * https://github.com/aiven/tsc-output-parser/blob/master/src/parserTypes.ts
+ *
+ * and from manually inspecting the output of `tsc-output-parser`.
+ *
+ * Just so we have at least a little structure for TypeScript to grab on to
+ * for the files we're `JSON.parse`-ing!
+ */
+interface TSError {
+  type: 'Item';
+  value: {
+    path: {
+      type: 'Path';
+      /**
+       * The path where the error occured
+       */
+      value: string;
+    };
+    cursor: {
+      type: 'Cursor';
+      value: {
+        line: number;
+        col: number;
+      };
+    };
+    tsError: {
+      type: 'TsError';
+      value: {
+        type: 'error';
+        /**
+         * These strings are TS error codes like `TS2339`
+         * see {@link https://github.com/microsoft/TypeScript/blob/main/src/compiler/diagnosticMessages.json}
+         */
+        errorString: string; // t
```

Comment:
Is 't' meant to stand for something, or is this a dangling comment?

Reply:
oops!